.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -33,3 +33,16 @@ steps:
     msbuildArgs: ${{ parameters.additionalBuildArguments }}
     clean: true
     maximumCpuCount: true
+
```

Comment:
@enricogior Since there were issues with other unit tests projects as well so I have enabled the VSTest task in the CI currently only for Preview Pane Unit tests assemblies for now. Let me know if this okay or we can just enable the CI for all tests later on.

Reply:
@udit3333 
actually it's a good idea to enable it for group of tests, so we can work incrementally reach the point when all the tests will pass.